<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="5" hostname="LAPTOP-4TQR0IMD" failures="1" tests="1" name="Test(failed)" time="0.041" errors="0" timestamp="2023-06-17T14:26:18 IST">
  <testcase classname="testNGxml.FailTestExecution" name="myMethod2" time="0.009">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNGxml.FailTestExecution.myMethod2(FailTestExecution.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- myMethod2 -->
  <testcase classname="testNGxml.FailTestExecution" name="myMethod4" time="0.0">
    <ignored/>
  </testcase> <!-- myMethod4 -->
  <testcase classname="testNGxml.FailTestExecution" name="myMethod3" time="0.0">
    <ignored/>
  </testcase> <!-- myMethod3 -->
  <testcase classname="testNGxml.FailTestExecution" name="myMethod1" time="0.0">
    <ignored/>
  </testcase> <!-- myMethod1 -->
  <testcase classname="testNGxml.FailTestExecution" name="myMethod6" time="0.0">
    <ignored/>
  </testcase> <!-- myMethod6 -->
  <testcase classname="testNGxml.FailTestExecution" name="myMethod5" time="0.0">
    <ignored/>
  </testcase> <!-- myMethod5 -->
</testsuite> <!-- Test(failed) -->
